scripts/scil_labelseg.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+
+"""
+LabelSeg: automatic tract labelling without tractography.
```

Comment:
we should come up with a better name _maybe_ scil_fodf_labelseg.py

Reply:
Still open to suggestions ! I've stuck with labelseg so far only because I haven't gotten a better alternative.